inbox/models/when.py

```diff
@@ -97,7 +97,7 @@ def parse(cls, raw):
         # the singular form if they are equal.
         start, end = cls.parse_keys(raw)
         if start > end:
-            raise ValueError("'{}' must be < '{}'.".format(*cls.json_keys))
+            start, end = (end, start)
```

Comment:
Why fix this? We shouldn't try to second-guess API users. Also, do you have a link to the sentry trace --- I couldn't find it anywhere.